google-cloud-notification/pom.xml

```diff
@@ -0,0 +1,198 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>google-cloud-notification</artifactId>
+  <version>0.26.1-alpha-SNAPSHOT</version>
+  <packaging>jar</packaging>
+  <name>Google Cloud Pub/Sub Notifications for Google Cloud Storage</name>
+  <url>https://github.com/GoogleCloudPlatform/google-cloud-java/tree/master/google-cloud-notification
+  </url>
+  <description>
+    Java idiomatic client for Google Cloud Notification.
+  </description>
+  <parent>
+    <groupId>com.google.cloud</groupId>
+    <artifactId>google-cloud-pom</artifactId>
+    <version>0.26.1-alpha-SNAPSHOT</version>
+  </parent>
+  <properties>
+    <site.installationModule>google-cloud-notification</site.installationModule>
+    <artifact.version>${project.version}</artifact.version>
+  </properties>
+  <dependencies>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-tcnative-boringssl-static</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>google-cloud-core</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>google-cloud-storage</artifactId>
+      <version>${storage.version}</version>
```

Comment:
Versions shouldn't be explicit here - all version lines should be removed so that they are inherited from dependencyManagement in the parent pom.